pydocxs3upload/image_upload.py

```diff
@@ -100,13 +122,20 @@ def upload(self, img_data, filename, image_format=None):
             message = error.find("Message").text
 
             raise ImageUploadException("S3 {0} - {1}".format(code, message))
-        elif r.status_code != self.OK_STATUS:
+        elif r.status_code not in self.OK_STATUS:
             raise ImageUploadException("S3 Upload Error: {0}".format(r.text))
```

Comment:
NON-BLOCKING: Should we specify in the exception message whether it was an error from GCS vs S3? (right now, it would look like it was S3 in either case, right?).

Reply:
I was trying to be surgical with these changes. How about I post a follow up PR that cleans up the messaging?